components/rpc-openstack.yml

```diff
@@ -8,6 +8,8 @@ name: rpc-openstack
 releases:
 - series: queens
   versions:
+  - sha: aae8ad6f1c09630fbf6d44d7028e8f02053b0f3a
```

Comment:
FYI this is not the head of the branch as it stood last night, so a full MNAIO deploy is now running because of that. The last PM build SHA can be seen in the job parameters: https://rpc.jenkins.cit.rackspace.net/view/RPC-O/job/PM_rpc-openstack-queens-xenial_mnaio_no_artifacts-swift-deploy/114/parameters/